internal/csot_util.go

```diff
@@ -32,3 +32,26 @@ func MakeTimeoutContext(ctx context.Context, to time.Duration) (context.Context,
 func IsTimeoutContext(ctx context.Context) bool {
 	return ctx.Value(timeoutKey{}) != nil
 }
+
+// TestRTTMonitor implements the RTTMonitor interface and is used internally for testing.
+type TestRTTMonitor struct{}
```

Comment:
Optional: Consider calling this something like a `ZeroRTTMonitor` or `NoopRTTMonitor` to indicate it basically always returns 0.